scripts/copy-released-manifests.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -e
+
+SCRIPT_DIR="$(dirname "$(readlink -f "$0")")"
+REPO_DIR="$(realpath "${SCRIPT_DIR}/..")"
+
+source "${SCRIPT_DIR}/common.sh"
+
+visit "${REPO_DIR}/tasks"
+    for TASK_NAME in *; do
+        mkdir -p "../released-tasks/${TASK_NAME}/${RELEASE_VERSION}"
```

Comment:
IMO change `released-tasks` to `release`. ArtifactHub URL is already long so let's make it shorter.